Few changes

MicrocontrollersDev opened the pull request:
- Capitalization changes
- Update Custom Crosshair to the new version released this year (I should do this on mine too...)
- Remove redundant "1.8.9" on TcpNoDelay and change description
- Add thing to ctjs saying to use 1.3.1 and not the legacy 0.18.4 (as lots of people get this wrong for some reason)
- Add orange simple mods to bad

LunaNotdev commented:
(1.8.9) is because the actual mod is not useless and works but the 1.8.9 version of it which is bullshit doesn't work which is why I added that

MicrocontrollersDev commented:
> (1.8.9) is because the actual mod is not useless and works but the 1.8.9 version of it which is bullshit doesn't work which is why I added that

makes sense, added it back
